src/app/collective-rewards/rewards/backers/context/BackerRewardsContex.tsx

```diff
@@ -98,7 +98,7 @@ export const BackerRewardsContextProvider: FC<BackerRewardsProviderProps> = ({
       const rifEarned = calculateTotalEarned(rifRewards.earned)
       const rbtcEarned = calculateTotalEarned(rbtcRewards.earned)
 
-      return rifEarned > 0n || rbtcEarned > 0n
+      return rifEarned > 0n && rbtcEarned > 0n
```

Comment:
Why do we need this to be in && rather than ||? Since that is used to claim a single token it may give wrong results